tmail-backend/rate-limiter/src/main/scala/com/linagora/tmail/rate/limiter/api/RateLimitation.scala

```diff
@@ -0,0 +1,83 @@
+package com.linagora.tmail.rate.limiter.api
+
+import com.linagora.tmail.rate.limiter.api.RateLimitation.AllowedQuantity
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Positive
+
+import java.time.Duration
+import java.util.UUID
+import scala.util.Try
+
+object RateLimitation {
+  type AllowedQuantity = Long Refined Positive
```

Comment:
Please reuse `org.apache.james.rate.limiter.api.AllowedQuantity`

Reply:
TODO Here, 
I commented TODO in source code, I will update it when updating James project